services/rollout-service/Makefile

```diff
@@ -0,0 +1,88 @@
+# This file is part of kuberpult.
+
+# Kuberpult is free software: you can redistribute it and/or modify
+# it under the terms of the Expat(MIT) License as published by
+# the Free Software Foundation.
+
+# Kuberpult is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+# Copyright 2023 freiheit.com
+
+
+include ../../Makefile.variables
+
+MAKEFLAGS += --no-builtin-rules
+
+export CGO_ENABLED=0
+
+GO := go
+
+GO_FILES := $(shell find . -type f -name '*.go')
+
+
+IMAGENAME?=$(IMAGE_REGISTRY)/kuberpult-rollout-service:${VERSION}
+
+GARBAGE := *.sentinel
+
+proto:
+	$(MAKE) -C../../pkg/api all
+
+bin/main: $(GO_FILES) | proto
```

Comment:
please make `bin/main` also dependent on the go files in /pkg, like here: 
https://github.com/freiheit-com/kuberpult/blob/main/services/cd-service/Makefile#L83